lib/internal/bootstrap_node.js

```diff
@@ -246,13 +243,44 @@
     perf.markMilestone(NODE_PERFORMANCE_MILESTONE_BOOTSTRAP_COMPLETE);
   }
 
+  function pushValueToArray() {
+    for (var i = 0; i < arguments.length; i++)
+      this.push(arguments[i]);
+  }
+
+  function setupProcessBinding(bindingObj) {
+    const _binding = process.binding;
+
+    process.binding = function binding(module) {
+      if (!module || typeof module !== 'string')
+        return;
```

Comment:
It’s unfortunate, but `process.binding` is used in the wild too often for us to be able to just return undefined if the requested module is undefined. We should cast it to a string just as the C++ version did.

Reply:
Also, when would core code call it with something that is not a string?  This guard should probably just go.